docs/src/pages/components/typography/TypographyTheme.tsx

```diff
@@ -1,22 +1,16 @@
 import * as React from 'react';
-import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
-
-const useStyles = makeStyles((theme: Theme) =>
-  createStyles({
-    root: {
-      ...theme.typography.button,
```

Comment:
We should retain this, using Box defeats the purpose of the demo. Please use the styled API instead.

Reply:
@oliviertassinari 

```tsx
const Component = styled('div')(({ theme }) => ({
  ...theme.typography.button,
  backgroundColor: theme.palette.background.paper,
  padding: theme.spacing(1),
}));
```

The above throws the following error

![image](https://user-images.githubusercontent.com/57921784/109803706-22c34980-7c00-11eb-809c-1e930834b5b6.png)

Reply:
I have faced the same issue when implementing the branding about page. The solution I have found:

https://github.com/mui-org/material-ui/blob/0e199f591021455edd1bf3eece2a71609a94e158/docs/src/modules/branding/BrandingHeader.tsx#L86

We track the issue in #22208.

Reply:
I have faced the same issue when implementing the branding about page. The solution I have found:

https://github.com/mui-org/material-ui/blob/0e199f591021455edd1bf3eece2a71609a94e158/docs/src/modules/branding/BrandingHeader.tsx#L86

We track the issue in #22208.

Reply:
Thanks for hack (: